dotnet-desktop-guide/framework/wpf/graphics-multimedia/how-to-control-key-frame-animation-timing.md

```diff
@@ -26,7 +27,7 @@ The following example uses a <xref:System.Windows.Media.Animation.DoubleAnimatio
 
 The following illustration shows when the value of each key frame is reached.
 
-![Key values are reached at 3, 4, and 10 seconds](./media/graphicsmm-keyframe-keytime1-timespan.png "graphicsmm_keyframe_keytime1_timespan")
+![Illustration showing four key frames with an increasing blue gradient color having the values of 0, 3, 4, and 10 seconds.](./media/graphicsmm-keyframe-keytime1-timespan.png "graphicsmm_keyframe_keytime1_timespan")
```

Comment:
Even though the surrounding text says, "illustration," the alt-text should still be "Screenshot, showing..."